### PR TITLE
Add 'prefix' variable

### PR DESCRIPTION
**Pull request by pr33thi**
Just trying to spread this out into as many PRs as possible (aka I didn't commit this change the first time around and didn't realize)
@orangejenny

### CODE REVIEW

File: corehq/apps/export/static/export/js/download_export.ng.js [hunk 1]
```diff
@@ -151,7 +151,8 @@
                         + getFilterName($scope.exportList[0].export_type) + "]")
                         .val().split(','),
                     function (item) {
-                        if (item.substring(0,3) === "t__") {
+                        var prefix = "t__";
+                        if (item.substring(prefix.length) === prefix) {
                             return userTypes[item.substring(3)];
```
Comment (orangejenny) [inline corehq/apps/export/static/export/js/download_export.ng.js]:
```suggestion
                            return userTypes[item.substring(prefix.length)];
```

**Review by orangejenny — APPROVED** [review]

File: corehq/apps/export/static/export/js/download_export.ng.js (hunk 1 above)
Comment (pr33thi) [reply corehq/apps/export/static/export/js/download_export.ng.js]: Thanks, and glad I got to use the sweet new github feature

Comment (orangejenny) [reply corehq/apps/export/static/export/js/download_export.ng.js]: apologies for mucking up the spacing

Comment (pr33thi) [reply corehq/apps/export/static/export/js/download_export.ng.js]: no prob! https://github.com/dimagi/commcare-hq/pull/22140/commits/ce0ee9c2c0e0322fb5273122f54915acef270c3b

**Pull request merged**